### DISCUSSION TITLE
Components within Tables - delegate to parent?

### DISCUSSION THREAD

**Discussion by rotorbit69** [livewire/livewire#3691] [1. Help]
I am testing livewire actions -I have a livewire component, with a toggle button, and then loop through 'rows' with another livewire component which also has an toggle button.
i.e
Livewire-component1 ... [button1]
foreach
Livewire component2 [button2]
Livewire component2 [button2]
Livewire component2 [button2]

Click the button1 - hands control to Livewire-component1
Click the button2 - hands control to Livewire-component2

Perfect!

However, configure as a table, and place <table> html tags around the foreach and now all actions are processed by Livewire-component1 ?

Why does 'table' stuff it up?

**Comment by joshhanley** [comment]
@rotorbit69 are you able to share some code? As I can't tell exactly what you mean 🙂 

But generally, if actions that are inside a child component are getting picked up by the parent, that suggests that there isn't one root element in the child component view. So make sure there is only one root element in the children.

Hope this helps!
